Tools/scripts/verify_ensurepip_wheels.py

```diff
@@ -0,0 +1,96 @@
+#! /usr/bin/env python3
+
+"""
+Compare checksums for wheels in :mod:`ensurepip` against the Cheeseshop.
+
+When GitHub Actions executes the script, output is formatted accordingly.
+https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-a-notice-message
+"""
+
+import hashlib
+import json
+import os
+import re
+import sys
+from pathlib import Path
+from urllib.request import urlopen
+
+PACKAGE_NAMES = ("pip", "setuptools")
+ENSURE_PIP_ROOT = Path(__file__).parent.parent.parent / "Lib/ensurepip"
+WHEEL_DIR = ENSURE_PIP_ROOT / "_bundled"
+ENSURE_PIP_INIT_PY_TEXT = (ENSURE_PIP_ROOT / "__init__.py").read_text(encoding="utf-8")
+GITHUB_ACTIONS = os.getenv("GITHUB_ACTIONS")
```

Comment:
For explicitness?
```suggestion
GITHUB_ACTIONS = os.getenv("GITHUB_ACTIONS", False)
```

Reply:
I believe checking its presence is enough. On GitHub Actions the value will be `"true"` as a string. I don't think it is worth changing it to `GITHUB_ACTIONS = os.getenv("GITHUB_ACTIONS") == "true"` either.

Reply:
In that case, maybe `"GITHUB_ACTIONS" in os.environ` -- makes it clear that we don't care about the value at all.

A

Reply:
On the second thought, I changed it to `GITHUB_ACTIONS = os.getenv("GITHUB_ACTIONS") == "true"` to be explicit.

Reply:
Sorry, missed this comment -- I still think the simpler check is better.

A